wunderio/client-UK-FSA-beta
# FSA-1027 - Webform submission edit form is simplified

Ticket: https://wunder.atlassian.net/browse/FSA-1027

Test: https://fsa.dev.wunder.io/admin/reports/send-to-la-failures

This is the second take of this feature and should be carefully tested on stage before going into production.

This change adds the following features:
- Module `fsa_webform` is added to provide various webform related features.
- `Save` button is added to webform submission edit form which allows administrators to easily save the form without going through the pages of the webform. 
- `Failed to send` view is updated with operation links which simplifies the overview of the listing.

Steps to review:
1. Run `drush cim -y` and `drush cr`
2. Go to `https://fsa.dev.wunder.io/admin/reports/send-to-la-failures` and see that `Edit` buttons are visible for submissions
3. Click on `Edit` and see that `Continue` or `Previous` are not visible and `Save` button is present
4. Find the email of a local authority and save the form; see that you are taken back to the list of submissions
